packages/hash/api/src/graphql/resolvers/knowledge/page/page.ts

```diff
@@ -72,11 +72,11 @@ export const pages: ResolverFn<
 > = async (_, { ownedById }, { dataSources: { graphApi }, userModel }) => {
   const accountModel = ownedById
     ? await UserModel.getUserById(graphApi, {
-        entityId: userAccountIdToEntityId(ownedById),
+        entityId: `${systemOrgAccountId}%${ownedById}`,
       }).catch((error: Error) => {
         if (error instanceof EntityTypeMismatchError) {
           return OrgModel.getOrgById(graphApi, {
-            entityId: userAccountIdToEntityId(ownedById),
+            entityId: `${systemOrgAccountId}%${ownedById}`,
```

Comment:
As above, we should be using this instead: https://github.com/hashintel/hash/blob/88e84beca68a835394f270350bb6d9e59e7aa62b/packages/hash/subgraph/src/types/identifier.ts#L14

Reply:
https://github.com/hashintel/hash/pull/1596/commits/6e6c358e80915bc8fb3fb132d66f3382fbfaf395